src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -631,5 +635,7 @@ public Cancellable rolloverIndex(RolloverRequest rolloverRequest, ActionListener
         public Cancellable getAliases(GetAliasesRequest getAliasesRequest, ActionListener<GetAliasesResponse> listener) {
             return this.indicesClient.getAliasAsync(getAliasesRequest, RequestOptions.DEFAULT, listener);
         }
+
```

Comment:
@kokibas can we remove this extra line here

Reply:
@owaiskazi19  yes it added by spotless